k8s/migration/internal/controller/migrationplan_controller.go

```diff
@@ -1497,3 +1509,71 @@ func (r *MigrationPlanReconciler) migrateRDMdisks(ctx context.Context, migration
 	}
 	return nil
 }
+
+// validates that the VM has a valid OS type
+func (r *MigrationPlanReconciler) validateVMOS(vmMachine *vjailbreakv1alpha1.VMwareMachine) (bool, bool, error) {
+	validOSTypes := []string{"windowsGuest", "linuxGuest"}
```

Comment:
Can we double check if these are `windowsGuest` and `linuxGuest` or just `windows` and `linux`